app/controllers/mentor/applications_controller.rb

```diff
@@ -8,6 +8,20 @@ def show
     @comment     = application.find_or_initialize_comment_by(current_user)
   end
 
```

Comment:
Yes, it's just a personal note for the mentor. It's already in production and this PR doesn't target the commenting function. I'm not too invested in whether it's gonna be note or comment. Comment has the bonus of riding on already existing logic.